test/controllers/admin/items/holds/positions_controller_test.rb

```diff
@@ -0,0 +1,34 @@
+require "test_helper"
+
+module Admin
+  module Items
+    module Holds
+      class PositionsControllerTest < ActionDispatch::IntegrationTest
+        include Devise::Test::IntegrationHelpers
+
+        setup do
+          @user = create(:admin_user)
+          create(:verified_member, user: @user)
+
+          sign_in @user
+        end
+
+        test "updates positions of holds" do
+          item = create(:item)
+          10.times.map { create(:hold, item: item) }
```

Comment:
Totally optional micro-nit: [`Integer#times` takes a block directly](https://ruby-doc.org/3.2.2/Integer.html#method-i-times), so `map` is not needed here as you're not using the Array it builds and returns.